### DISCUSSION TITLE
Dynamically update an open Bootstrap 5 modal backdrop and keyboard properties

### DISCUSSION THREAD

**Discussion by kainers1** [twbs/bootstrap#37692] [Q&A]
Hello community,

I have a Bootstrap 5 modal that is displayed. I want to dynamically change it's backdrop and keyboard-options so that for a specific action i.e. esc-button press, the modal will stay static.

With v4 i have used
```
$('#modalsubmit').on("click", function(e) {
   $("#myModal").data('bs.modal')._config.backdrop = 'static'; // or true
   $("#myModal").data('bs.modal')._config.keyboard = false; // or true
   // ... do something
});  
```
This is not working with v5 and I have no idea what to change. Thanks

**Comment by kainers1** [comment]
Any idea?

**Comment by martinbean** [comment]
@mdo Any idea on this one? As I have the same requirement.

I have a form within a modal, and I want to prevent the modal from being dismissed only whilst a form submission is in flight.
